apps/checkout/src/hooks/useErrorMessages.ts

```diff
@@ -1,10 +1,33 @@
+import { ValidationErrorCode } from "@/lib/globalTypes";
 import { useFormattedMessages } from "./useFormattedMessages";
 
 export const useErrorMessages = () => {
   const formatMessage = useFormattedMessages();
 
-  return {
+  const errorMessages = {
     invalidValue: formatMessage("invalid"),
     requiredField: formatMessage("required"),
+    requiredValue: formatMessage("required"),
```

Comment:
These 2 seem to be identical in meaning to the end user. I don't think we should be making distinction between "required" and "missing"

Reply:
Leftover, the function already wouldn't consider "missing" but forgot to delete it altogether